source/content/nutzen/api-nutzen.rst

```diff
@@ -1,154 +1,473 @@
+   * - issued
+     - Das Erstelldatum der Ressource.
+   * - id
+     - Die ID der Ressource.
+   * - download_url
+     - Die Adresse, unter der die Ressource heruntergeladen werden kann.
+   * - media-type
+     - Der Medientyp der Ressource.
+   * - format
+     - Das Format der Ressource.
+   * - rights
+     - Die Nutzungsbedingungen der Ressource.
+   * - created
+     - Der Zeitpunkt, als die Ressource in der Datenbank eingerichtet wurde.
+   * - description
+     - Die Beschreibung der Ressource.
+   * - num_resources
+     - Die Anzahl Ressourcen für das Dataset.
+
 
 .. _api-nutzen-terminologie:
 
 Terminologie von CKAN in opendata.swiss
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Alle Objekte von opendata.swiss sind auch in CKAN vertreten. Damit der API-Zugriff reibungslos
-funktioniert, erklären wir Ihnen in der folgenden Auflistung, mit welcher Terminologie Sie welche
-Einheiten in opendata.swiss erreichen können:
+Über die API können Sie nach sämtlichen Metadaten in opendata.swiss suchen. Die Suchsyntax entspricht Apache `Lucene <https://lucene.apache.org/>`__, da opendata.swiss auf CKAN basiert und Apache `Solr <https://solr.apache.org/>`__ als Suchmaschine benutzt. Dies ist ebenfalls in der  `CKAN-Dokumentation <https://docs.ckan.org/en/latest/user-guide.html#search-in-detail>`__ ausführlich beschrieben. 
+Arbeiten Sie mit Metadaten der Organisation, Metadaten der Datasets oder Metadaten der Ressourcen? Beachten Sie dabei die Ebenen: Die folgende Auflistung zeigt die wichtigsten Terminologien, mit welchen Sie Informationen in opendata.swiss finden können:
+
 
+``organization``
+   Entspricht der `datenpublizierenden Organisation <https://opendata.swiss/de/organization>`__. Jedes Dataset wird von genau einer Organisation publiziert. 
+   Eine Organisation kann jedoch Unterorganisationen haben.
 ``package``, ``dataset``
    Entspricht einem :term:`Dataset <Dataset>` bei opendata.swiss. Damit gemeint sind die Metadaten
    zu einem Bündel von Ressourcen.
-``resource``
-   Entspricht einer Ressource bei Opendata.swiss. Die Plattform opendata.swiss hostet
-   selbst keinen Ressourcen, stellt aber «downloadUrls» oder «accesssUrls»
-   und teilweise auch Daten Previews zur Verfügung.
-   Ressourcen in CKAN entsprechen dcat:Distribution im :doc:`DCAT AP Standard <../glossar/bibliothek/dcat-ap-ch>`
+``id``
+   Entspricht in der Regel dem :term:`Slug <Slug>` des Datasets oder der Organisation auf opendata.swiss. 
+   Bei Ausnahmefällen konsultieren Sie bitte die CKAN-Dokumentation.
+``resource``, ``distribution``
+   Entspricht einer `Ressource <https://handbook.opendata.swiss/de/content/glossar/begriffe.html#term-Distribution>`__ bei Opendata.swiss. Die Plattform opendata.swiss hostet selbst keine Datasets oder Ressourcen, stellt aber downloadUrls oder accesssUrls und teilweise auch Daten Previews zur Verfügung. Ressourcen in CKAN entsprechen dcat:Distribution im :doc:`DCAT AP Standard <../glossar/bibliothek/dcat-ap-ch>`.
```

Comment:
one could be a bit confused, you are given "Resource" link (`Ressource <https://handbook.opendata.swiss/de/content/glossar/begriffe.html#term-Distribution>`) but you are landing on the "Distribution" and only after reading "Distribution" text one can understand why: (Eine Distribution (oder auch «Ressource»))